src/app/organization/[orgSlug]/admin/settings/page.tsx

```diff
@@ -0,0 +1,36 @@
+import { Stack, Text, Title, Breadcrumbs, Divider } from '@mantine/core'
+import { Link } from '@/components/links'
+import { RequireOrgAdmin } from '@/components/require-org-admin'
+import { OrganizationSettingsManager } from './organization-settings-manager'
+import { getOrgFromSlugAction } from '@/server/actions/org.actions'
+
+export const dynamic = 'force-dynamic'
+
+export default async function AdminSettingsPage({ params }: { params: Promise<{ orgSlug: string }> }) {
+    const { orgSlug } = await params
+    // fetch the org on the server so the client can hydrate without a Loader
+    const initialOrg = await getOrgFromSlugAction(orgSlug)
+
+    const items = [
+        <Link href={`/organization/${orgSlug}/admin`} key="1">
+            Dashboard
+        </Link>,
+        <Text key="2">Admin</Text>,
+        <Text key="3" aria-current="page">
+            Settings
+        </Text>,
+    ]
+
+    return (
+        <Stack p="md">
+            <RequireOrgAdmin />
+            <Breadcrumbs>{items}</Breadcrumbs>
+            <Divider />
+            <Title order={1} mb="xl">
+                Settings
+            </Title>
+
+            <OrganizationSettingsManager orgSlug={orgSlug} initialOrg={initialOrg} />
```

Comment:
why is the edit and display components loading the org again instead of using  `initialOrg`?

Reply:
display is updating org data in background but I'm removing that functionality.